mock/local-journal.json

```diff
@@ -16,7 +16,7 @@
     {
       "slotDetail": {
         "slotId": 8165,
-        "start": "2019-01-25T08:00:00+00:00",
+        "start": "2019-01-28T08:00:00+00:00",
```

Comment:
It can cause confusion so easily

Reply:
This is true with the current strategy where we base the newly generated journal on the old one that's checked into the repo, but I was thinking more about generating the whole journal programatically with a reasonable spread of slots across days. You're more familiar with the script, is there something making this more difficult than it sounds?

Reply:
Would be very difficult to assign slightly meaningful data to all the journal slot attributes randomly no? You need that data to come from somewhere static. We could potentially create the date programatically.

It really pollutes the PRs, because every day is a new `local-journal.json` gets generated which will be pushed up to the repo. We could have a brainstorming session around this topic?